components/brand/Testimonials.vue

```diff
@@ -288,4 +286,12 @@ onMounted(() => {
   transform: translateX(20px);
   opacity: 0;
 }
+.image-container{
+  @media (min-width: 1024px) {
+    height: 250px; 
+    width: 400px;
+  }
+    height: 170px; 
+    width: 320px;
+}
```

Comment:
This is forcing the image to have dimensions `320px`x`170px` **always**, not only for mobile views.

However, the parent component is hidden on large screens by using this class: `"grid grid-cols-1 items-center justify-center block lg:hidden mx-auto mt-8"`, and therefore this `div` will never reach `1024px`.

I would suggest removing that breakpoint.

```suggestion
.image-container {
    height: 170px; 
    width: 320px;
}
```